Update eq-translations to 4.9.2

rmccar opened the pull request:
### What is the context of this PR?
This updates the eq-translations dependancy to version 4.9.2 and recreates the lockfile. There was a warning when recreating the lockfile and to fix that I have also added a package folder which is what has been done in some other repos to solve this.

### How to review
- Tests pass

### Checklist
* [ ] eq-translations updated to support any new schema keys which need translation

rmccar commented:
> We should probably mention actions update in PR description or ideally in the title (if we want to keep these actions updates in this PR since it seems irrelevant), also, don't main and release yamls need checkout updates as well?

Ive added more information the description on that change and also added the updates for the main and release workflows

petechd commented:
> > We should probably mention actions update in PR description or ideally in the title (if we want to keep these actions updates in this PR since it seems irrelevant), also, don't main and release yamls need checkout updates as well?
> 
> Ive added more information the description on that change and also added the updates for the main and release workflows. But I can take the change out if wanted. I updated them to see if it helped with the warnings/errors I was seeing and just left them in because I thought it would be good to update them anyway but I understand that is a separate change really.

 Probably worth taking it out since there is a card for it: https://jira.ons.gov.uk/browse/ECI-1263

rmccar commented:
> > > We should probably mention actions update in PR description or ideally in the title (if we want to keep these actions updates in this PR since it seems irrelevant), also, don't main and release yamls need checkout updates as well?
> > 
> > 
> > Ive added more information the description on that change and also added the updates for the main and release workflows. But I can take the change out if wanted. I updated them to see if it helped with the warnings/errors I was seeing and just left them in because I thought it would be good to update them anyway but I understand that is a separate change really.
> 
> Probably worth taking it out since there is a card for it: [jira.ons.gov.uk/browse/ECI-1263](https://jira.ons.gov.uk/browse/ECI-1263)


> > > We should probably mention actions update in PR description or ideally in the title (if we want to keep these actions updates in this PR since it seems irrelevant), also, don't main and release yamls need checkout updates as well?
> > 
> > 
> > Ive added more information the description on that change and also added the updates for the main and release workflows. But I can take the change out if wanted. I updated them to see if it helped with the warnings/errors I was seeing and just left them in because I thought it would be good to update them anyway but I understand that is a separate change really.
> 
> Probably worth taking it out since there is a card for it: [jira.ons.gov.uk/browse/ECI-1263](https://jira.ons.gov.uk/browse/ECI-1263)

Id probably leave it in as the work is already done and it reduces the work needed for that card but ok Ill take it out. Ill also mention the need for checkout to be updated on that card
